### PR TITLE
fix Closing parent in Treeview closes descendants, but "arrow" icon is not changed properly

### PR DESCRIPTION
**Pull request by mostafahesham-dev**

### CODE REVIEW

File: build/js/Treeview.js [hunk 1]
```diff
@@ -92,7 +93,6 @@ class Treeview {
     const $parent = $relativeTarget.parent()
 
     let treeviewMenu = $parent.find(`> ${SELECTOR_TREEVIEW_MENU}`)
-
```
Comment (danny007in) [inline build/js/Treeview.js:95]: dont delete this line

Comment (mostafahesham-dev) [reply build/js/Treeview.js:95]: done

**Comment by mostafahesham-dev** [comment]
@REJack done

File: build/js/Treeview.js [hunk 2]
```diff
@@ -92,7 +93,7 @@ class Treeview {
...
+
```
Comment (danny007in) [inline build/js/Treeview.js:96]: `npm run js-lint`
and resolve it
because now there is whitespace

**Pull request merged**